src/lib/swapper/swappers/OneInchSwapper/buildTrade/buildTrade.ts

```diff
@@ -2,10 +2,10 @@ import { fromAssetId, fromChainId } from '@shapeshiftoss/caip'
 import type { EvmChainId } from '@shapeshiftoss/chain-adapters'
 import type { Result } from '@sniptt/monads'
 import { Err, Ok } from '@sniptt/monads'
-import { DAO_TREASURY_ETHEREUM_MAINNET } from 'constants/treasury'
 import { bnOrZero } from 'lib/bignumber/bignumber'
 import type { BuildTradeInput, SwapErrorRight } from 'lib/swapper/api'
 import { makeSwapErrorRight, SwapErrorType } from 'lib/swapper/api'
+import { getTreasuryAddressForReceiveAsset } from 'lib/swapper/swappers/ZrxSwapper/utils/helpers/helpers'
```

Comment:
want to move this to `utils/evm/index.ts` now that it is shared, or if we won't have any need outside of swapper then `swappers/utils/helpers` would be fine

Reply:
Good catch, right you are!

Reply:
https://github.com/shapeshift/web/pull/4818/commits/6a34edfa056e29bf0e152865a94f63049e1c8e97